trin-execution/src/storage/evm_db.rs

```diff
@@ -86,17 +86,29 @@ impl EvmDB {
         }
     }
 
-    pub fn get_storage_trie_diff(&self, address_hash: B256) -> BrownHashMap<B256, Vec<u8>> {
+    pub fn get_storage_trie_diff(
+        &self,
+        storage_root: B256,
+        address_hash: B256,
+    ) -> BrownHashMap<B256, Vec<u8>> {
         let mut trie_diff = BrownHashMap::new();
 
+        let account_db = AccountDB::new(address_hash, self.db.clone());
+        let trie = if storage_root == keccak256([EMPTY_STRING_CODE]) {
+            EthTrie::new(Arc::new(account_db))
```

Comment:
Can we just return empty result here?

If that's the case, I would say you can revert most of the changes in this file? Or am I missing something?

Reply:
This no longer applies I will click resolve, but feel free to bring it up again if you still think there is a issue